adal/tests/Test.ADAL.NET.UIAutomation/Infrastructure/KeyVaultSecretsProvider.cs

```diff
@@ -54,12 +56,12 @@ public class KeyVaultSecretsProvider
         ///         "clientId": [client ID]
         /// </para>
         /// </remarks>
-        /// <param name="section">Configuration section for Key Vault</param>
         public KeyVaultSecretsProvider()
         {
             _config = new KeyVaultConfiguration();
-            _config.AuthType = KeyVaultAuthenticationType.UserCredential;
+            _config.AuthType = KeyVaultAuthenticationType.ClientCertificate;
```

Comment:
has info on how to place the cert in the right location been added to the "how to setup a new build/test machine"?

Reply:
Yes